iib/workers/tasks/build.py

```diff
@@ -756,6 +790,14 @@ def _update_index_image_build_state(request_id, prebuild_info):
     if from_index_resolved:
         payload['from_index_resolved'] = from_index_resolved
 
+    source_from_index_resolved = prebuild_info.get('source_from_index_resolved')
+    if source_from_index_resolved:
+        payload['source_from_index_resolved'] = source_from_index_resolved
+
+    target_from_index_resolved = prebuild_info.get('target_from_index_resolved')
```

Comment:
Same, could you please rename `target_from_index_resolved` to `target_index_resolved`?

Reply:
done